src/app/components/source/SourcePicture.js

```diff
@@ -154,13 +115,22 @@ class SourcePicture extends Component {
 
   render() {
     return (
-      <StyledImage
+      <div
         alt="avatar"
         size={this.props.size}
         type={this.props.type}
-        className={this.props.className}
+        className={cx(
+          styles['source-picture'],
+          {
+            [this.props.className]: true,
+            [styles.sizeLarge]: this.props.size === 'large',
```

Comment:
This is more of a question than a review comment, but how do you approach style class naming? I'm not always sure when to name my class using camelCase, PascalCase or kebab-case

Reply:
yeah, honestly its something we/I am not consistent about, but, but maybe as a front-end team we can come to some sort of standard.
Random not complete thoughts that don't lead to any conclusion, but 🤷 :
- sometimes when doing refactoring I maintain "whatever was there before" to keep the PR clearer
- camelCase is more like our js, so sometimes it more closely matches the other parts of the code
- kebab-case is more of a css standard approach that most people prefer/linters